apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.scss

```diff
@@ -1,3 +1,4 @@
+@import "../../../../assets/css/color.scss";
 p.bankName {
```

Comment:
p.bankName class name should be .bank-name

Reply:
fixed